examples/peripherals/lcd/i80_controller/main/i80_controller_example_main.c

```diff
@@ -177,19 +187,44 @@ void app_main(void)
         .bits_per_pixel = 16,
     };
     ESP_ERROR_CHECK(esp_lcd_new_panel_nt35510(io_handle, &panel_config, &panel_handle));
+#elif CONFIG_EXAMPLE_LCD_I80_CONTROLLER_ILI9341
+    // ILI9341 is NOT a distinct driver, but a special case of ST7789
+    // (essential registers are identical). A few lines further down in this code,
+    // it's shown how to issue additional device-specific commands.
+    ESP_LOGI(TAG, "Install LCD driver of ili9341 (st7789 compatible)");
+    esp_lcd_panel_dev_config_t panel_config = {
+        .reset_gpio_num = EXAMPLE_PIN_NUM_RST,
+        .color_space = ESP_LCD_COLOR_SPACE_BGR,
+        .bits_per_pixel = 16,
+    };
+    ESP_ERROR_CHECK(esp_lcd_new_panel_st7789(io_handle, &panel_config, &panel_handle));
 #endif
 
     esp_lcd_panel_reset(panel_handle);
     esp_lcd_panel_init(panel_handle);
     // Set inversion, x/y coordinate order, x/y mirror according to your LCD module spec
+    // the gap is LCD panel specific, even panels with the same driver IC, can have different gap value
 #if CONFIG_EXAMPLE_LCD_I80_CONTROLLER_ST7789
     esp_lcd_panel_invert_color(panel_handle, true);
+    esp_lcd_panel_set_gap(panel_handle, 0, 20);
 #elif CONFIG_EXAMPLE_LCD_I80_CONTROLLER_NT35510
     esp_lcd_panel_swap_xy(panel_handle, true);
     esp_lcd_panel_mirror(panel_handle, true, false);
-#endif
-    // the gap is LCD panel specific, even panels with the same driver IC, can have different gap value
     esp_lcd_panel_set_gap(panel_handle, 0, 20);
```

Comment:
it's was a mistake made by me, in fact, I didn't see an NT35510 screen that needs an extra gap setting. So would you please help me remove this line or just set it to zero gap?